webpack.config.js

```diff
@@ -1,10 +1,10 @@
 const webpack = require('webpack');
 const path = require('path');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
-const uglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
 module.exports = {
+  mode: "development",
```

Comment:
Does this need to be removed?

Reply:
Yeah, potentially. I am not completely sure about what webpack is expecting us to do here. What I _do_ know is that, if you remove this line, Webpack will throw a warning.